pootle/apps/pootle_profile/templatetags/profile_tags.py

```diff
@@ -87,17 +88,27 @@ def profile_social(profile):
 
 
 @register.inclusion_tag("user/includes/profile_teams.html")
-def profile_teams(profile):
+def profile_teams(request, profile):
```

Comment:
yes, in order to restrict what is viewable - that or permissions, but for now it only restricts anon so the request.user is kinda needed